### DISCUSSION TITLE
Wait Update,, CasaOS on OpenWRT

### DISCUSSION THREAD

**Discussion by ZYXCXDX** [IceWhaleTech/CasaOS#257] [Q&A]
When will CasaOS be stable and installable on OpenWrt,,.?

**Comment by Hedius** [comment]
Afaik OpenWRT is already supported (see the readme of the repo)

You will have to manually install docker first.

**Reply by ZYXCXDX** [reply]
Yup, but in my case {OpenWrt 21.02 for aarch64/arm64 (Amlogic-s9xxx) }
Can't be installed with issue OS Not recognized,
N ofcourse already installed Docker before...
